src/drivers/dw/dma.c

```diff
@@ -336,6 +337,67 @@ static inline void dw_update_bits(struct dma *dma, uint32_t reg, uint32_t mask,
 	io_reg_update_bits(dma_base(dma) + reg, mask, value);
 }
 
+static inline void dw_dma_interrupt_mask(struct dma *dma, int channel)
+{
+	struct dma_pdata *p = dma_get_drvdata(dma);
+
+	if (!p->chan[channel].irq_enabled) {
+		tracev_event(TRACE_CLASS_DMA, "dw_dma_interrupt_mask(): "
+			     "dma %d channel %d not working in irq mode",
```

Comment:
for the grep's sake, let try to avoid breaking tracing strings. I'd rather split it after "TRACE_CLASS_DMA," and maybe remove "working" to make it a bit shorter.
Also, consider what happens if this condition is met - IRQ isn't enabled and verbose logging is on, you'd get these messages in every call - to DMA start, stop, interrupt register, unregister,... Maybe just select one location to print this?

Reply:
But it's not like it is traced in copy happening every 1 ms. It's still will be limited number of traces. If you enable verbose you need to live with more messages.